Update the Tutorials timeline for 17 weeks

sbis04 opened the pull request:
Have diluted the timeline to `17 weeks` and updated the **Tutorials** blog post.

Rohitesh-Kumar-Jain commented:
Thanks for updating, seems good to me.

Rohitesh-Kumar-Jain commented on jekyll/_posts/2021-05-11-gsod-2021-team-proposal-tutorials.md:
```diff
@@ -71,6 +71,10 @@ The overall structure of the Tutorials will be as follows:
+  
+  A guide for installing the development tools required for running Wechaty bot on any user system.
+
 * **Quick start**
```
LGTM timeline

proudofsimin commented:
It looks great! Thank you for your effort!

huan reviewed (approved):
LGTM

Please get approvals from all volunteers and your team mates to make this PR fulfill the requirements of merging.

Thank you very much!

sbis04 commented:
@proudofsimin & @shwetalsoni if this PR is alright can you please approve it, so that it fulfills the requirements of merging.

sbis04 commented:
As @proudofsimin had requested to list down the changes in the updated timeline for easier understanding, there are as follows:

* We have added a section called **Installation** page to our overall tutorial structure, and added it under our week 2 tasks.
* Instead of keeping the "Usage with Docker" and "Usage with Heroku" under **Tutorials**, it is to be included under **How-to guides** (only links for those pages will be under tutorials so that users can quickly navigate to them if they are searching for it, this resolves the conflict between the tutorials and the how-to guide sections).
* Remaining things are almost the same, just diluted the timeline to 17 weeks, @proudofsimin you can update them on your docs (where you are tracking the completion of tasks w.r.t. PRs).

Rohitesh-Kumar-Jain commented:
ping @huan

Rohitesh-Kumar-Jain commented:
@huan can we merge this pls? It will be great if the updated proposal is merged : )
